geoips/plugins/modules/algorithms/RGB_Default.py

```diff
@@ -0,0 +1,59 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Data manipulation steps for "Night_Vis_IR" product.
+
+This algorithm expects two VIIRS channels (DNBRad and M16BT) for a RGB image
+"""
```

Comment:
Should edit this docstring to indicate that this is an unused algorithm rather than the Night_Vis_IR product.

Reply:
See #561.